### ISSUE TITLE
[Feature] Add an option to change "Paste As Plain Text" as the default.

### ISSUE THREAD

**Issue by flymin** [typora/typora-issues#4721]
I know currently, we have different shortcuts for "Paste" and "Paste As Plain Text". However, I think it would be much convenient to have an option to change the default behavior.

**What I want to do is to use cmd+V to paste as plain text.**
However, if I set it like this:
![image](https://user-images.githubusercontent.com/28671430/137108059-6abd740c-07e3-488b-9f7f-95e1d712e0c4.png)
This "Edit" drop-down will be like this:
![image](https://user-images.githubusercontent.com/28671430/137108239-bc0b5245-8693-4f2c-bd94-542ae30c16ed.png)

If I change the name to "Paste as Plain Text" (a lower case of "a" here), the "Edit" drop-down will show the original shortcut for "Paste as Plain Text".

Therefore, I suppose I cannot set Typora to "Paste as Plain Text" with cmd+V. I do not know if there is any bug here (Version 0.11.11 (5873)).

However, since we already have an option for "Copy", why not add similar option for "Paste"?

**Comment by GreatestFool** [comment]
I have to agree with this. I'm encountering a similar problem.

When registering a [custom keybind](https://support.typora.io/Shortcut-Keys/#change-shortcut-keys) for `Paste` and `Paste as Plain Text` in v1.4.8 by inverting their keybinds, it doesn't actually work. The [menu changes its presented keybind](https://user-images.githubusercontent.com/3003484/204882943-a49cb4d7-2986-4789-8ede-83454ae9da2d.png), but when actually using the keybinds, nothing has changed. E.g.

```json
"keyBinding": {
    "Paste": "Ctrl+Shift+V",
    "Paste as Plain Text": "Ctrl+V"
}
```

Will still result in Ctrl + V pasting with formatting instead of as plain text. Likewise, Ctrl + Shift + V pastes as plain text and not with formatting.

**Comment by abnerlee** [comment]
fixed key binding in v1.5.x
